doc/user/evpn.rst

```diff
@@ -502,3 +502,11 @@ the creation of the VLAN subinterfaces and directly enslave "eth0" to "br10".
       ip link set eth0 master br10
 
 This completes the necessary configuration for an L2VNI.
+
+Displaying evpn mac vni information
+-------------------------------------
+
+.. clicmd:: show evpn mac vni " CMD_VNI_RANGE " detail [json]
```

Comment:
Can this be defined a bit like for an _operator-readable-style_?

Reply:
I meant: `(1-16777215)`.

Reply:
Fixed it